bedrock/firefox/views.py

```diff
@@ -284,7 +284,7 @@ class LatestFxView(TemplateView):
     """
     non_fx_redirect = 'firefox.new'
 
-    @vary_on_headers('User-Agent')
+    @cache_control(max_age=0)
```

Comment:
calling `patch_cache_control` is what the `cache_control` decorator does too. They seem identical really, just that the `cache_control` one is more about setting it for some > 0 time.